webui/templates/home.html

```diff
@@ -2,7 +2,9 @@
 
 {% block body %}
 <!-- jumbotron -->
-  <div class="jumbotron" id="jumbotron"></div>
+  <div class="jumbotron bg-nd-blue d-flex justify-content-center" id="jumbotron">
+  <img class="img-fluid" src="../static/banner-1300.jpg" alt="">
```

Comment:
use the `{{ url_for("static", ...... }}` thing here instead of hardcoding a path to the banner.